.rspec

```diff
@@ -0,0 +1,2 @@
+--require spec_helper
```

Comment:
Why do you require `spec_helper` and not `rails_helper` Please research the difference between those 2 files.

Reply:
Upon researching, I've found that rails_helper is the exact same thing just reworked to accomplish two things; to keep the installation process in sync with regular RSpec changes 
and to provide an out-of-the-box way to avoid loading Rails for those specs that do
not require it. In my own words, spec_helper is specific to tests that don't need rails. Rails incorporates the spec_helper and transforms it into a universal testing environment without having to modify the surrounding files to require both. Should we be changing it?

Reply:
For those reasons you should include rails helper here and omit the require in spec files